docs/faq.md

```diff
@@ -48,3 +48,23 @@ New password for default administrator (user-xxxxx):
 ### I added an additional disk with partitions. Why is it not getting detected?
 
 As of Harvester v1.0.2, we no longer support adding additional partitioned disks, so be sure to delete all partitions first (e.g., using `fdisk`).
+
+### Why are there some Harvester pods that become ErrImagePull/ImagePullBackOff?
+
+This is likely because your Harvester cluster is an air-gapped setup, and some pre-loaded container images are missing. Kubernetes has a mechanism that does garbage collection against bloated image stores. When the partition which stores container images is over 85% full, `kubelet` will try to prune some least used images to save disk space until the occupancy is lower than 80%. These numbers (85% and 80%) are default High/Low thresholds that come with Kubernetes.
```

Comment:
```suggestion
If Harvester pods show `ErrImagePull` or `ImagePullBackOff`, your Harvester cluster is likely an air-gapped setup, and some pre-loaded container images are missing. Kubernetes has a mechanism that does garbage collection against bloated image stores. When the partition storing container images is over 85% full, `kubelet` tries to prune images that are no longer being used to save disk space until the occupancy of the image store is lower than 80%. The 85% and 80% numbers are the default High/Low thresholds in Kubernetes.
```

Reply:
Per the sentence

> ... tries to prune images that are no longer being used ...

And the paragraph from the [Kubernetes documentation page](https://kubernetes.io/docs/concepts/architecture/garbage-collection/#container-image-lifecycle)

> Disk usage above the configured `HighThresholdPercent` value triggers garbage collection, which deletes images in order based on the last time they were used, starting with the oldest first.

An image may still be being used but pruned by kubelet. I suggest going with the official docs, i.e., changing the sentence from "no longer being used ..." to "the images based on the last time they were used, starting with the oldest," to clarify.

```suggestion
This is likely because your Harvester cluster is an air-gapped setup, and some pre-loaded container images are missing. Kubernetes has a mechanism that does garbage collection against bloated image stores. When the partition which stores container images is over 85% full, `kubelet` tries to prune the images based on the last time they were used, starting with the oldest, until the occupancy is lower than 80%. These numbers (85% and 80%) are default High/Low thresholds that come with Kubernetes.
```

What do you think?

Reply:
@starbops Agreed, want to the official K8s docs. Thanks for calling this out.